sklearn/model_selection/_split.py

```diff
@@ -994,16 +994,16 @@ class ShuffleSplit(BaseShuffleSplit):
 
     Parameters
     ----------
-    n_splits : int (default 10)
+    n_splits : int, default 10
         Number of re-shuffling & splitting iterations.
 
-    test_size : float, int, or None, default 0.1
-        If float, should be between 0.0 and 1.0 and represent the
-        proportion of the dataset to include in the test split. If
-        int, represents the absolute number of test samples. If None,
-        the value is automatically set to the complement of the train size.
+    test_size : float, int, None, or 'default', default 'default'
+        If float, should be between 0.0 and 1.0 and represent the proportion
+        of the dataset to include in the test split. If int, represents the
+        absolute number of test samples. If None, the value is set to the
+        complement of the train size. If 'default', the value is set to 0.1.
```

Comment:
"By default".

Then describe 'The default will change in version ... : it will remain 0.1 only if train_size is unspecified, otherwise it will complement. The specified train_size."